runtime/doc/filetype.txt

```diff
@@ -663,6 +663,19 @@ To disable this behavior, set the following variable in your vimrc: >
 
 	let g:python_recommended_style = 0
 
+QUERY					                *ft-query-plugin*
+
+
+By default enables linting of tree-sitter queries for installed parsers using
+|lua-treesitter-query_linter|. Query linting can be disabled using >vim
+
+	autocmd FileType query let b:lint_on = []
+<
+The events that trigger linting are by default `InsertLeave`, `CursorHoldI`
+and `BufEnter`. To change the events that trigger linting to other events
+like `BufWrite` and `BufEnter`, use >vim
+
+	autocmd FileType query let b:lint_on = ["BufWrite", "BufEnter"]
```

Comment:
This doesn't work for me since the `ftplugin` is triggered on `FileType` which will get called before any user ones.

Reply:
I think `b:lint_on` should be `g:query_lint_on`

Reply:
Aaand we're back full circle...